components/EditOutivity/index.js

```diff
@@ -0,0 +1,156 @@
+import styled from "styled-components";
+import { useState } from "react";
+
+export default function EditOutivityForm({
+  onEditOutivity,
+  outivity,
+  setIsEdit,
+}) {
+  const [editedOutivity, setEditedOutivity] = useState({
+    id: outivity.id,
+    title: outivity.title,
+    area: outivity.area,
+    country: outivity.country,
+    image: outivity.image,
+    description: outivity.description,
+  });
+
+  const saveEdits = (event) => {
+    event.preventDefault();
+    onEditOutivity(editedOutivity);
+    setIsEdit(false);
+  };
+
+  return (
+    <StyledNewOutivitiesForm onSubmit={saveEdits}>
+      <h1>Edit Outivity</h1>
+      <StyledNewOutivitiesFormFields>
+        <StyledNewOutivitiesFormField>
+          <label htmlFor="outivityImage">Image</label>
+
+          <StyledNewOutivitiesFormInput
+            type="text"
+            name="outivityImage"
```

Comment:
When you give your inputs the same names as the keys in your outivity object, the data-object in the submit handler will have the exact key-value pairs.